contrib/guix/Dockerfile

```
@@ -15,12 +15,12 @@ ARG guix_checksum_aarch64=72d807392889919940b7ec9632c45a259555e6b0942ea7bfd13110
 ARG guix_checksum_x86_64=236ca7c9c5958b1f396c2924fcc5bc9d6fdebcb1b4cf3c7c6d46d4bf660ed9c9
 ARG builder_count=32
 
-ENV PATH /root/.config/guix/current/bin:$PATH
+ENV PATH=/root/.config/guix/current/bin:$PATH
```

let's add "" also?
```suggestion
ENV PATH="/root/.config/guix/current/bin:$PATH"
```

because previous PATH can have not only alpha-numeric characters

---

makes sense to me